overlay/etc/dnsmasq.d/steamcache.conf

```diff
@@ -0,0 +1,55 @@
+# Run as root to make things easy.
+user=root
+group=root
+
+# Add Google's public DNS servers as a fallback.
+# Docker seems to use these in /etc/resolv.conf anyway.
+server=8.8.8.8
+server=8.8.4.4
+
+# Docker adds some weird stuff to /etc/hosts.
+no-hosts
+
+# Cache any local IPs for an hour.
+# This is 0 by default, but one lost packet causes Windows to switch DNS.
+local-ttl=3600
```

Comment:
I confirmed with Wireshark that `address` lines use this value, otherwise they default to a TTL of 0. That was a fun bug to track down. I'll update the wording of the comment.